docs/releasing_openttd.md

```diff
@@ -8,8 +8,8 @@ This guide is for OpenTTD developers/maintainers, to release a new version of Op
 
 * If this is an RC1 (first Release Candidate) build, create a new branch `release/nn` where `nn` is the major version number, then apply changes similar to [PR#9573](https://github.com/OpenTTD/OpenTTD/pull/9573). You also need to forwardport the changelog, as in [PR#10113](https://github.com/OpenTTD/OpenTTD/pull/10113).
   * Update the version in `CMakeLists.txt` in the master branch, heading for the next major release, e.g. from 14.0 to 15.0.
-  * Add a new (empty) AI compatibility script in `bin/ai/`
-  * Add the new version to CheckAPIVersion in `src/ai/ai_info.cpp` and `src/game/game_info.cpp`
+  * Add a new (empty) AI compatibility script in `bin/ai/` and `bin/game/`
```

Comment:
```suggestion
  * Add a new (empty) AI compatibility script in `bin/ai/` and `bin/game/` for the version of the branch
```
Or something along those lines, as you wouldn't want it to be the same version you add to `ApiVersions`. Maybe move it towards the saveload.h line as that is also about the branched of version, and not where master is heading to.